OSX builds are failing with 
```
##[warning]An image label with the label MacOS does not exist.
,##[error]The remote provider was unable to process the request.
```

--- a/eng/jobs/osx-build.yml
+++ b/eng/jobs/osx-build.yml
@@ -4,19 +4,16 @@ parameters:
 jobs:
 - job: ${{ parameters.name }}
   pool:
-    ${{ if eq(variables['System.TeamProject'], 'public') }}:
-      name: Hosted macOS
-    ${{ if ne(variables['System.TeamProject'], 'public') }}:
-      vmImage: macOS-latest
+      vmImage: 'macOS-latest'
   strategy:
-    matrix: 
+    matrix:
       debug:
         _BuildConfig: Debug
       release:
         _BuildConfig: Release
   workspace:
     clean: all
-  variables: 
+  variables:
     CommonMSBuildArgs: >-
       /p:Configuration=$(_BuildConfig)
       /p:PortableBuild=true
@@ -40,7 +37,7 @@ jobs:
       /p:OfficialBuildId=$(OfficialBuildId)
       /p:StripSymbols=true
       $(CommonMSBuildArgs)
-    displayName: Build 
+    displayName: Build
     condition: succeeded()
 
   - template: steps/upload-job-artifacts.yml

